compute_tools/src/compute.rs

```diff
@@ -59,6 +59,8 @@ pub static PG_PID: AtomicU32 = AtomicU32::new(0);
 
 /// Compute node info shared across several `compute_ctl` threads.
 pub struct ComputeNode {
+    /// The ID of the compute
+    pub id: String,
```

Comment:
minor suggestion:

```suggestion
    pub compute_id: String,
```
Yeah, can tell that it's a compute ID because the struct is called ComputeNode, but plain `id` is such a generic name that I think it's better to always spell it out in full.

Reply:
That makes sense. Also, more greppable.